scripts/generate-sdk.js

```diff
@@ -11,19 +11,16 @@ if (!fs.existsSync(serviceRoot)) {
 }
 
 const sources = findLatestSources(serviceRoot)
+  .filter(source => /acm/.test(source))
```

Comment:
So I disabled generating all modules, because I found it easier to iterate on a single module. Some of the other modules won't build now, because I haven't handled all of the types